LiskHQ/lisk-desktop
# Users should be able to see every delegates locked balance - Closes #4006

### What was the problem?

This PR resolves #4006

### How was it solved?

- Changed the locked balance logic to also calculate locked balances for other delegates.
- Restrict unlocking balances to only from the wallet page.

### How was it tested?

- Open Lisk Desktop
- Visit a delegate page
- You should be able to see their locked balance (but not be able to unlock it) if they have any.
- If you're on your wallet page, you should see an arrow beside your locked balance (if you have any)
- Clicking on the arrow should open a dialog to view more details and unlock your balance.
